corehq/sql_accessors/migrations/0029_get_case_ids_in_domain.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.form_processor.models import CaseTransaction
+from corehq.sql_db.operations import RawSQLMigration, HqRunSQL
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {
+    'TRANSACTION_TYPE_FORM': CaseTransaction.TYPE_FORM
+})
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0028_rename_get_multiple_forms_attachments'),
+    ]
+
+    operations = [
+        migrator.get_migration('get_case_ids_in_domain_1.sql'),
+        HqRunSQL(
```

Comment:
(copying comment so it shows up in the diff) I think there's one more function that should also get deleted though